features/achordion.c

```diff
@@ -108,6 +105,28 @@ bool process_achordion(uint16_t keycode, keyrecord_t* record) {
       (record->event.key.row < 254 && record->event.key.col < 254);
 #endif
 
+#ifdef ACHORDION_STREAK
+  if (achordion_state == STATE_RELEASED || achordion_state == STATE_TAPPING) {
+    if (is_tap_hold && record->tap.count == 0 && record->event.pressed &&
+        is_key_event) {
+      // A tap-hold key is pressed and considered by QMK as "held".
+      const uint16_t s_timeout = achordion_streak_timeout(keycode);
+      // Apparently I don't know how timer_expired wortks. Can't get it to work here.
+      //if (streak_timer && !timer_expired(record->event.time, streak_timer + 1900)) {
+      if (s_timeout && streak_timer && record->event.time < (streak_timer + s_timeout)) {
```

Comment:
Somehow this was triggering occasionally even if the timeout was 0. I added the `s_timeout &&` to prevent that.